src/main/kotlin/io/emeraldpay/dshackle/upstream/calls/DefaultEthereumMethods.kt

```diff
@@ -348,4 +346,30 @@ class DefaultEthereumMethods(
     override fun getSupportedMethods(): Set<String> {
         return allowedMethods.plus(hardcodedMethods).toSortedSet()
     }
+
+    class HardcodedData(netVersion: String, chainId: String) {
+        val netVersion: String
+        val chainId: String
+
+        init {
+            this.netVersion = netVersion.lowercase()
+            this.chainId = chainId.lowercase()
+        }
+
+        override fun equals(other: Any?): Boolean {
```

Comment:
use data class please :(

Reply:
Can't use of data class because I want to modify params not having extra fields